.gitignore

```diff
@@ -106,15 +139,33 @@ dmypy.json
 
 # End of https://www.gitignore.io/api/flask
 
-## Intellij ###
-.idea/
-
 ### macOS ###
 .DS_Store
 
 ### Virtual environment ###
 env/
-.venv/
 
 ### Configuration files ###
+Vagrantfile
+.git 
+Dockerfile
```

Comment:
We definitely don't want to git ignore the Dockerfile/Vagrantfile - it looks like you get away with these because they were already checked in before being added here, but we do want to track those with our codebase